bot/__main__.py

```diff
@@ -16,7 +18,12 @@
 
 sentry_sdk.init(
     dsn=Client.sentry_dsn,
-    integrations=[sentry_logging]
+    integrations=[
+        sentry_logging,
+        RedisIntegration(),
+        AioHttpIntegration()
+    ],
+    release=f"pydis-sir-lancebot@{GIT_SHA}"
```

Comment:
This can just be `sir-lancebot`.
```suggestion
    release=f"sir-lancebot@{GIT_SHA}"
```